app/models/message.rb

```diff
@@ -53,6 +55,16 @@ def reply?
     @reply || false
   end
 
+  # TODO: Revist realtionship between Messages, replies, drafts, and replydrafts
+  def as_replydraft
+    @replydraft = true
+    self
+  end
+
+  def replydraft?
+    @replydraft || false
```

Comment:
I don't think these methods should be on Message, they should be on MessageDraft instead. And maybe message draft skips validations or we build new inheritance hirearchy for validaitons, but this is really confusing to have these methods here as opposed to having htem on Message Draft I think, no? Maybe we get rid of MessageDraft altogether, and then it makes sense I suppose.